nebula-docker-compose/docker-compose-ssl.yaml

```diff
@@ -6,9 +6,9 @@ services:
       USER: root
       TZ:   "${TZ}"
     command:
-      - --meta_server_addrs=metad0:45500,metad1:45500,metad2:45500
-      - --local_ip=metad0
-      - --ws_ip=metad0
+      - --meta_server_addrs=172.28.1.1:45500,172.28.1.2:45500,172.28.1.3:45500
```

Comment:
why change theses addresses

Reply:
> why change theses addresses

Because  only the meta leader can return correct response. For client, it does not know the leader address, so client random chose one meta server addrs, when this addr is not leader, then client get the real leader from response and re-connect and re-execute. 
So we should expose all the meta server address, or it will throw error for `connection refused`